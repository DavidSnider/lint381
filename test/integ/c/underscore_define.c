#define FOO
#define __FOO__
