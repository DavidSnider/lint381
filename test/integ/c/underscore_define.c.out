underscore_define.c:2:1: error: Macro '__FOO__' should not start with an underscore
